aviary/mission/gasp_based/ode/flight_path_eom.py

```diff
@@ -56,7 +57,7 @@ def setup(self):
             add_aviary_input(self, Dynamic.Mission.ALTITUDE, val=np.ones(nn), units="ft")
             add_aviary_input(self, Dynamic.Mission.DISTANCE, val=np.ones(nn), units="ft")
 
-        self.add_output("TAS_rate", val=np.ones(nn), desc="TAS rate", units="ft/s**2",
+        self.add_output(Dynamic.Mission.VELOCITY_RATE, val=np.ones(nn), desc="TAS rate", units="ft/s**2",
                         tags=['dymos.state_rate_source:TAS', 'dymos.state_units:kn'])
```

Comment:
This tag looks wrong now. it's referring to dymos.state_rate_source:TAS, but we're actually calling it Velocity_rate now. Not sure if this will brick something later.